### PR TITLE
Muskingum-Cunge simple application notebook

### PR DESCRIPTION
**Pull request by awlostowski-noaa**
I created a new jupyter notebook illustrating the application of the Muskingum-Cunge Fortran routine. The notebook reviews seral core topics
-  compiling and importing Fortran extension module with f2py
- executing the Fortran model in the context of two simple numerical experiments

## Additions
- One new jupyter notebook

## Removals
- None

## Changes
- None

## Testing
1.  Repeated successful notebook execution on my local machine. Reviewer should attempt to run the notebook on their end to confirm. 

## Screenshots
image

## Notes
- The f2py3 stuff may no longer be needed once @groutr changes are included in the base repo

## Todos
- Review
- Maybe add more complicated examples as needed.

### CODE REVIEW

File: notebooks/mc-wave-routing-demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
...jZ6aCewNbKSW5yCE/fKJDWUSi6FvWFcJjCX6Q20G8BDRnb7nAM+H7X8KXFYdG9Hdx7k16hwGzIgt/xa4NMznEA1NsT/wDyA7lP8+fAb9iEZJHVbjs/st8MMwfzIwK8z/iGiIm07hONYRDe19WPgcuwLdie5GvjHss5Jdd133jMV5KfDbdH+nfGq+yZuqXGuVTFPVBYqGms8ieqDQGKIhGiAaYqT69Zdh/h3gEUlPA9UDB34JOCjWf9EDGEX04KcnzawKWCnp9T04lsVmNgdA0lzgNTMzSXOIEkt1HGfH+gs6A0OJnm9RbSBQElt+D7hV0mDgb2b2iaQvEv24T1f0QLcuRIMhHgW8ZdHzIzCz6meTfIHoLAYze11SH0ndw7rJFg2mVyZpDdEw7scBz1lodpI0KRbPbOBxSc8Dz8fK1xAlX9dOeOJwbZKkYcCNwOFmtkHSI0Q/ttWs5ryZXSPpSKIB4WZIOozoL///NLOXatR/Bs2nLDafiC0n2PV/UMB5Zragnnq2EztGM3tC0vtEx/NiaGYT8GczuyW+o6Qv72HcVTT8e3EmUcL9MlFCG2vRcyw6h9hdO+F9HK6t6g5sAzZJGkD0vJK4C2Ov7wFIGmFm75vZ7UR/uQ8BXgK+HYYGR9JoSbnAW8CFoQ9kIHBSio/nJeA/FU4TJB1SyzYL2XWGgqThQLGZ/Qb4O9HAeK8B50vqH7bpLWkfYCpwfEi4SOodqvk3UVMSkk4E1lqNZ3/U8BZwrqQukroRJQkkZQBDzOwNomdE9ADywj6jiQbTc+2En3G41qqLoqefVfuXmd1cvWBmH0qaCXxM1Hb/To39e0maTfRX88Wh7J7QgSyiH9gPiZpX8oEPwo92CdEjNJ8javOfBywlJJ8U+gnRMNmzw4/wYmp0KJvZNkmLJI00syLgAuBrkiqIngj4UzNbL+k2oqdGZhCNBnytmU0NzXp/C+VrgFOJ+jIeCp9VKXBFfUGa2QeSniL67NYA08OqTOAxST2IPt/fWPQcCYiS7i2fq8y1WT46rmt3JC0h6kBfm+5YmpukrwCHmdlt6Y4lGeFs8Akz+2K6Y3HNx884nGtDzOw5SX3SHUcjDAW+l+4gXPPyMw7nnHON4p3jzjnnGsUTh3POuUbxxOGcc65RPHE455xrFE8czjnnGuX/AwDnii5/KLS9AAAAAElFTkSuQmCC\n",
```
Comment (jameshalgren) [inline notebooks/mc-wave-routing-demo.ipynb:138]: Usually, we'd want to store this without the image information embedded in the notebook. @awlostowski-noaa Let's talk about what makes most sense.

Comment (awlostowski-noaa) [reply notebooks/mc-wave-routing-demo.ipynb]: Noted. I am going to get into the practice of saving notebooks after I have cleared all outputs.

**Comment by awlostowski-noaa** [comment]
@jameshalgren have you tested this on  Colab? If so, and if successful, this notebook should be ready to go.

**Comment by awlostowski-noaa** [comment]
Blacked version looks good. I'll make sure to incorporate that formatting AND the Google Colab requirements into my workflow. My most recent commit also moved the basic system imports to the top of the document.

**Pull request merged**